bootstrap/cmd/bootstrap/app/kfctlClient.go

```diff
@@ -63,9 +64,20 @@ func NewKfctlClient(instance string) (KfctlService, error) {
 
 // CreateDeployment issues a CreateDeployment to the requested backend
 func (c *KfctlClient) CreateDeployment(ctx context.Context, req kfdefs.KfDef) (*kfdefs.KfDef, error) {
-	resp, err := c.createEndpoint(ctx, req)
-	if err != nil {
-		return nil, err
+	var resp interface{}
+	var err error
+	// Add retry logic
+	bo := backoff.WithMaxRetries(backoff.NewConstantBackOff(2*time.Second), 30)
```

Comment:
does this mean we retry this at most for 1 min?  what is the estimated maximum time to create an endpoint?

Reply:
Let me change it to 1 hour in case our cluster is in short of resources.